terraform/api_gateway.tf.json.template.py

```diff
@@ -116,6 +116,51 @@ def for_domain(cls, domain):
     'status': '$context.status'
 }
 
+file_download_limit = config.waf_file_download_limit
+if file_download_limit is None:
+    file_download_rate_rule = ()
+else:
+    window_length = 60 * 10
+    limit, concurrency = file_download_limit
+    file_download_rate_rule = [{
+        'name': 'FileDownloadRateLimit',
+        'action': {
```

Comment:
Generally, the most logical ordering of these keys is `statement` first, then `action` and visibility last. If existing code has a different ordering you could fix that in a separate commit. If that's anything but trivial, I'd be fine with just doing it for this rule.